### ISSUE TITLE
Remove `skip` tag from JS feature spec

### ISSUE THREAD

**Issue by kgf** [ReadyResponder/ReadyResponder#473]
This test is bypassed due to JS issues. When running this JS spec, we aren't logged in...

This should be tested first. Sean recently updated JS testing

https://github.com/ReadyResponder/ReadyResponder/blob/development/spec/features/events_spec.rb#L90

**Comment by ThomasCarlson** [comment]
Just tested on top of the trunk development.  still failing.

**Comment by kgf** [comment]
Thanks @ThomasCarlson

**Comment by goldenhub77** [comment]
@kgf @ThomasCarlson None of the JS testing changes I made were committed to this branch.

**Comment by kgf** [comment]
@goldenhub77 , once the icon is merged in, I think I will PR the changes from your feature branch into development. It is atomic and self contained enough. That will give @ThomasCarlson the changes you made in JS testing.

**Comment by goldenhub77** [comment]
@kgf @ThomasCarlson Since ui_v3 merge into dev this test is now passing.

**Comment by kgf** [comment]
That's great news. Thanks to all.
